apps/web-client/src/lib/components/supplier-orders/ComparisonTable.svelte

```diff
@@ -1,14 +1,20 @@
 <script lang="ts">
-	import { sortLinesBySupplier, type ProcessedOrderLine } from "$lib/db/cr-sqlite/utils";
+	import { sortLinesBySupplier } from "$lib/db/cr-sqlite/order-reconciliation";
+	import type { ProcessedOrderLine, SupplierPlacedOrderLine } from "$lib/db/cr-sqlite/types";
 
 	import type { BookEntry } from "@librocco/db";
+	import { onMount } from "svelte";
 
-	export let supplierBooks: ProcessedOrderLine[];
+	export let supplierBooks: { processedLines: ProcessedOrderLine[]; unmatchedBooks: (BookEntry & { quantity: number })[] } = {
```

Comment:
Might be better to rename it `reconciledBooks` or  `processedOrderLines` - something to relate it to the processing in the reconciliation flow

Reply:
?